MVVM-C/ViewModels/MVVMCDetailViewModel.swift

```diff
@@ -9,14 +9,13 @@
 import Foundation
 
 
-class MVVMCDetailViewModel: DetailViewModel
-{
+class MVVMCDetailViewModel: DetailViewModel {
     weak var viewDelegate: DetailViewModelViewDelegate?
     weak var coordinatorDelegate: DetailViewModelCoordinatorDelegate?
     
     fileprivate(set) var detail: DataItem? {
         didSet {
-            viewDelegate?.detailDidChange(viewModel: self)
+            viewDelegate?.detailDidChangeFor(viewModel: self)
```

Comment:
Maybe `detailsDidChange(for:)` will better fit Swift 3 proposed naming convention (see https://swift.org/documentation/api-design-guidelines/#naming ).